src/types/kana.tsx

```diff
@@ -0,0 +1,9 @@
+type KanaType = {
+  id: string;
+  pronunciation: string;
+  kana: string;
+  type: 'basic' | 'tenten' | 'yoon';
```

Comment:
I think this has nothing to do with typing. You schould use a namespace here and a typing for the namespace :)

Reply:
Can you elaborate more on that? I don't quite understand what you are suggesting :)

Reply:
I saw you build an enum, thats perfect :)